problem-14/src/app/component/comment-view/comment-view.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, DestroyRef, inject} from '@angular/core';
+import {CommentService} from "../../service/comment.service";
+import {takeUntilDestroyed} from "@angular/core/rxjs-interop";
+import {Comment} from "../../model/comment";
+
+@Component({
+  selector: 'app-comment-view',
+  standalone: true,
+  imports: [],
```

Comment:
no import needed?

Reply:
Just did a double check and no import is needed for this problem.